mackerel-plugin-gcp-compute-engine/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"github.com/littlekbt/mackerel-plugin-gcp-compute-engine/lib"
```

Comment:
Please change this import path. It should be `github.com/mackerelio/mackerel-agent-plugins/mackerel-plugn-gcp-compute-engine/lib`?